ProcessLib/BoundaryCondition/ConstraintDirichletBoundaryCondition.h

```diff
@@ -113,9 +113,10 @@ class ConstraintDirichletBoundaryCondition final : public BoundaryCondition
     MeshLib::Mesh const& _bulk_mesh;
 
     /// The function _getFlux calculates the flux through the boundary element.
-    std::function<Eigen::Vector3d(
-            std::size_t const, MathLib::Point3d const&, double const,
-            GlobalVector const&)> _getFlux;
+    std::function<Eigen::Vector3d(std::size_t const, MathLib::Point3d const&,
+                                  double const,
+                                  std::vector<GlobalVector*> const& x)>
```

Comment:
To stay consistent with the other parameters:
```suggestion
                                  std::vector<GlobalVector*> const&)>
```

Reply:
Or add to all the other parameters useful names.